spec/javascripts/actions/note_spec.js

```diff
@@ -35,6 +35,21 @@ describe('Note Actions', () => {
           done();
         });
     });
+
+    it('dispatches setLoadingStory when promise fails', () => {
```

Comment:
`done` is not being passed in the test function, we gotta add it otherwise it will fail after the test suit finishes.
```suggestion
    it('dispatches setLoadingStory when promise fails', (done) => {
```